### PR TITLE
Awesome Books Es6 Functionalities

### PR DESCRIPTION
**Pull request by Othniel2471**
### In this branch I implemented:

- Added HTML structure for the project.
- Added CSS for HTML structure.
- Created a module for the CRUD book functionalities.
- Created a module for the navigation functionality.
- Created a module for the date functionality.

### CODE REVIEW

File: assets/js/modules/books.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+      window.addEventListener('DOMContentLoaded', () => {
+        if (localStorage.getItem('books')) {
+          const allBooks = JSON.parse(localStorage.getItem('books'));
+          inv.displayBooks(allBooks);
+        }
+      });
+    }
+
+    // eslint-disable-next-line class-methods-use-this
```
Comment (t-yanick) [inline assets/js/modules/books.js]: - [ ] Great job working on this milestone. You should not disable eslint rules, but rather fix the errors raised. The linters are there to make us write robust codes. Kindly remove the disable rule.

File: assets/js/modules/books.js [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+       const bookDetails = new Books(title, author);
+
+       if (localStorage.getItem('books')) {
+         Books.books = JSON.parse(localStorage.getItem('books'));
+       }
+       Books.books.push(bookDetails);
+       localStorage.setItem('books', JSON.stringify(Books.books));
+     }
+
+     // eslint-disable-next-line class-methods-use-this
```
Comment (t-yanick) [inline assets/js/modules/books.js]: - [ ] Great job working on this milestone. You should not disable eslint rules, but rather fix the errors raised. The linters are there to make us write robust codes. Kindly remove the disable rule.

File: assets/js/modules/books.js [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+     clearField = () => {
+       Books.bookTitle.value = '';
+       Books.bookAuthor.value = '';
+     }
+
+     // eslint-disable-next-line class-methods-use-this
```
Comment (t-yanick) [inline assets/js/modules/books.js]: - [ ] Great job working on this milestone. You should not disable eslint rules, but rather fix the errors raised. The linters are there to make us write robust codes. Kindly remove the disable rule.

**Review by t-yanick — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED

Hi @Othniel2471 , 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Project setup correctly
:heavy_check_mark: App is working correctly
:heavy_check_mark: Effective use of ES6
:heavy_check_mark: Good commit messages

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Munsa1 — APPROVED** [review]
Hi @Othniel2471, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
congratulations-1

## Highlights
Made Requested Changes :heavy_check_mark: 
App Works As Expected :heavy_check_mark: 


## Optional suggestions

### Important :red_circle: 
- [x] Kindly use the correct github flow going forward for this module as this is a core requirement
image
Your github flow should be `feature-branch` asking to merge into `main` branch, not into `development` branch. Checkout this [resource](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md)

- [x] Two. Use the imperative tense when writing your commit messages and capitalize the first letter of the first word in your commit messages. Refer to this [resource](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)

- [x] Three. Use descriptive branch names for your feature branches as this is also a core requirement. refer to the link above.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**